roles/contiv_network/templates/aci_gw.j2

```diff
@@ -16,6 +16,8 @@ start)
     -e "APIC_PASSWORD={{ apic_password }}" \
     -e "APIC_LEAF_NODE={{ apic_leaf_nodes }}" \
     -e "APIC_PHYS_DOMAIN={{ apic_phys_domain }}" \
+    -e "APIC_EPG_BRIDGE_DOMAIN={{ apic_epg_bridge_domain }}" \
+    -e "APIC_CONTRACTS_UNRESTRICTED_MODE={{ apic_contracts_unrestricted_mode }}" \
```

Comment:
Yes, if there are good defaults we should define them.

This might be obvious but here are a few things to consider when adding defaults:
- a default value shall not cause bad things to happen in a users's setup. 
- if a variable is a must have for functionality to work and there is no good default. It is perfectly ok to let them remain undefined. This helps ansible fail early and errors are caught immediately.